docs/reference/virtual_memory_reference.rst

```diff
@@ -0,0 +1,13 @@
+.. meta::
+  :description: This chapter describes introduces Virtual Memory (VM) and shows
+                how to use it in AMD HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, virtual memory, virtual, memory, VM
+
+.. _virtual_memory_reference:
+
+*******************************************************************************
+HIP Virtual Memory Management API
+*******************************************************************************
+
+.. doxygengroup:: Virtual
+   :content-only:
```

Comment:
Noticed the following text in the documentation:

![image](https://github.com/ROCm/HIP/assets/164940351/eda63ecb-7311-49e9-9af9-a8c25def83d8)

This should be replaced everywhere with:
Note
This API is implemented on Linux and is under development on Microsoft Windows.
Warning
This API is marked as Beta. While this feature is complete, it can change and might have outstanding issues.

Reply:
Hi @lpaoletti 

I also found this problem, and it's appearing everywhere in the repo. Fixing this in a separate PR.

Reply:
https://github.com/ROCm/HIP/pull/3542

Reply:
It was fixed and merged in the mentioned PR.